library/src/main/res/values/attrs.xml

```diff
@@ -8,6 +8,10 @@
             <enum name="Grid" value="1"/>
             <enum name="LinearLayoutWithHeaders" value="2"/>
         </attr>
+        <attr name="rrvOrientation">
```

Comment:
Didn't want to use an orientation tied to LinearLayout because we can also use this for GridLayouts